src/pages/dashboard/dashboard.style.tsx

```diff
@@ -1,3 +1,18 @@
 import styled from '@emotion/styled'
 
-export const ContentStyled = styled('div')``
```

Comment:
these changes should not be here, you should use Main-Layout branch as your start point,